opencti-platform/opencti-graphql/src/database/middleware.js

```diff
@@ -3138,7 +3138,7 @@ const buildEntityData = async (context, user, input, type, opts = {}) => {
 const createEntityRaw = async (context, user, rawInput, type, opts = {}) => {
   // region confidence control
   const input = { ...rawInput };
-  const { confidenceLevelToApply } = controlCreateInputWithUserConfidence(user, input);
+  const { confidenceLevelToApply } = controlCreateInputWithUserConfidence(user, input, type);
```

Comment:
You have the same call in `createRelationRaw`, probably needs also the "type" argument to be consistent, even if we do not plan on adding confidence per relationship type.

As it seems, this argument can be made mandatory in `controlCreateInputWithUserConfidence`.